init.sqf

```diff
@@ -0,0 +1,8 @@
+#include "script_component.hpp"
+
+// environmentController eventHandlers
+[QGVAR(setTime), {skipTime ((_this select 0) - daytime + 24 ) % 24; forceWeatherChange;}] call CBA_fnc_addEventHandler;
```

Comment:
Might want to do this one multi-line since it has more than 1 line of code.
```sqf
[QGVAR(setTime), {
    skipTime ((_this select 0) - daytime + 24 ) % 24; // and add some comment here to explain what it is doing
    forceWeatherChange;
}] call CBA_fnc_addEventHandler;
```